pkgs/by-name/hd/hdr10plus/package.nix

```diff
@@ -23,12 +23,7 @@ rustPlatform.buildRustPackage (finalAttrs: {
     "dev"
   ];
 
-  inherit (hdr10plus_tool)
-    src
-    useFetchCargoVendor
-    cargoDeps
-    cargoHash
-    ;
+  inherit (hdr10plus_tool) src cargoDeps cargoHash;
```

Comment:
(Just for my sake: adding `useFetchCargoVendor` to `excludeDrvArgNames` makes it so that this doesn't eval, or was this done just as a general noop cleanup?)